hibernate-core/src/main/java/org/hibernate/type/descriptor/java/InstantJavaDescriptor.java

```diff
@@ -36,12 +36,12 @@ public InstantJavaDescriptor() {
 
 	@Override
 	public String toString(Instant value) {
-		return InstantType.FORMATTER.format( ZonedDateTime.ofInstant( value, ZoneId.of( "UTC" ) ) );
+		return DateTimeFormatter.ISO_INSTANT.format( value );
```

Comment:
`ISO_INSTANT` is also using UTC, so it's a good alternative to our previous formatting option.